src/otx/core/utils/build.py

```diff
@@ -14,19 +14,23 @@
     from omegaconf import DictConfig
     from torch import nn
 
-def build_mm_model(config: DictConfig, model_registry: Registry, load_from: str) -> nn.Module:
+def build_mm_model(config: DictConfig,
+                   model_registry: Registry,
+                   load_from: str,
+                   load_backbone_only: bool = False) -> nn.Module:
     """Build a model by using the registry."""
     from mmengine.runner import load_checkpoint
 
     from otx import algo  # noqa: F401
 
-
     try:
         model = model_registry.build(convert_conf_to_mmconfig_dict(config, to="tuple"))
     except AssertionError:
         model = model_registry.build(convert_conf_to_mmconfig_dict(config, to="list"))
 
-    if load_from is not None:
+    if load_backbone_only and load_from is not None:
```

Comment:
Why is `load_backbone_only` needed at this moment? I cannot find `load_backbone_only` presence in `recipe/segmentation/*.yaml`

Reply:
I got what your point. Please see my above comment.